### PR TITLE
Support tickets: webhooks

### PR DESCRIPTION
**Pull request by pzaichkina**
# Description

* Webhooks page updates based on support tickets
* Added SfE webhooks to the page

## Type of change

- [X] New document(s)/updating existing
- [ ] Fixes
- [ ] Styling
- [ ] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: docs/using-the-api/webhooks/core-rules-types.md [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
 | :- | :- | :- |
 | [Company data connection status changed](/using-the-api/webhooks/core-rules-types#company-data-connection-status-changed)  | A data connection's status changes. | `dataConnectionId`, `platformKey`, `newStatus`, `oldStatus` |
-| [New company synchronized](/using-the-api/webhooks/core-rules-types#new-company-synchronized)                | After the first dataType is successfully synced for a new company. | |
-| [Data sync completed](/using-the-api/webhooks/core-rules-types#data-sync-completed)                     | Data synchronization is completed; a notification will be generated for each `dataType` as the sync completes. | `dataType`, `datasetId` |
-| [Dataset data changed](/using-the-api/webhooks/core-rules-types#dataset-data-changed)                    | A dataset synchronization has completed and this has resulted in updates within Codat's data cache - this could be through the creation of new records or a change to existing records. | `dataType`, `datasetId` |
+| [New company synchronized](/using-the-api/webhooks/core-rules-types#new-company-synchronized)                | The first `dataType`` is successfully synced for a new company. | |
```
Comment (mcclowes) [inline docs/using-the-api/webhooks/core-rules-types.md:11]:
```suggestion
| [New company synchronized](/using-the-api/webhooks/core-rules-types#new-company-synchronized)                |  All datasets created during the initial sync of a company are completed.<br/><b>Legacy behaviour:<b> The first `dataType`` is successfully synced for a new company. [See deprecation](https://docs.codat.io/updates/231010-deprecation-webhooks-new-company-synchronized). | |
```

Comment (mcclowes) [reply docs/using-the-api/webhooks/core-rules-types.md:11]: Feel free to improve the language. See slack for more context

File: docs/using-the-api/webhooks/core-rules-types.md [hunk 2]
```diff
@@ -167,8 +170,8 @@ In line with industry standard security practices, we have removed personally id
 
 ### Account categories updated
 
-**Rule type:** `account-categories-updated`
-**Trigger:** Anytime that Codat updates the `suggested` fields or a user updates the `confirmed` fields.  
+**Rule type:** `account-categories-updated`
```
Comment (mcclowes) [inline docs/using-the-api/webhooks/core-rules-types.md:173]: I meant we need this for all webhooks

Comment (pzaichkina) [reply docs/using-the-api/webhooks/core-rules-types.md:173]: (this is an existing line of the doc) that was my question in the channel - could someone explain to me how I can find these?

**Review by dcoplowe — APPROVED** [review]

**Pull request merged**